uw-frame-components/staticFeeds/sample_notifications.json

```diff
@@ -1,31 +1,37 @@
 {"notifications" :
     [
       {
-        "id"     : 1,
-        "groups" : ["Portal Administrators"],
-        "title"  : "This is an admin notification smoke test",
-        "actionURL" : "http://www.google.com",
-        "actionAlt" : "Google",
-        "dismissable" : true,
-        "priority" : false
-      },
-      {
         "id"     : 2,
         "groups" : ["Users - Student SSN Update"],
         "title"  : "Our records indicate the Social Security Number in your Personal Information is invalid or missing. Please provide your Social Security Number or select to not provide your Social Security Number.",
         "actionURL" : "/portal/p/my-info-student-center-ssn.ctf1/max/action.uP?pP_action=loginAction",
         "actionAlt" : "UW-Madison Student Information",
         "dismissable" : true,
-        "priority" : false
+        "priority" : false,
+		"actionButtons" : []
```

Comment:
Are action buttons a required field for priority nots now? We should fail gracefully if it is null.